### PR TITLE
Level 1 Scene Layout

### LINKED ISSUE #1: Level 1 Scene Layout
AC1: Under the "Scenes" folder, add a new scene and name it "Level 1"
AC2: Add a tile map and populate the scene with tiles
AC3: Add a background
AC4: Add at least 20 coin animations throughout the level
AC5: Add a end level checkpoint "Flag"
AC6: Add 2D tilemap collider

### PR DESCRIPTION
**Pull request by shreyabhakta**

### CODE REVIEW

**Comment by jlfwilliams** [comment]
Booo, includes yaml files, please delete before i review

**Comment by shreyabhakta** [comment]
Lol maybe I SHOULD throw a few yaml files in there

File: New Unity Project/Assets/Animation/Chest.anim.meta [hunk 1]
```diff
@@ -0,0 +1,12 @@
+fileFormatVersion: 2
+<<<<<<< HEAD:New Unity Project/Assets/Animation/Chest.anim.meta
+guid: dd63bebfd6e7f451183ceb6f8b1e6304
+=======
+guid: c04beb24f38fa4a4e9e186939ed3e9bf
+>>>>>>> master:New Unity Project/Assets/Tiles/Asset 40.asset.meta
```
Comment (victoriametzler) [inline New Unity Project/Assets/Animation/Chest.anim.meta:6]: should this (">>>>>>> master:") be here?

File: New Unity Project/Assets/Animation/Chest.anim.meta [hunk 2]
```diff
@@ -0,0 +1,12 @@
+fileFormatVersion: 2
+<<<<<<< HEAD:New Unity Project/Assets/Animation/Chest.anim.meta
+guid: dd63bebfd6e7f451183ceb6f8b1e6304
+=======
```
Comment (victoriametzler) [inline New Unity Project/Assets/Animation/Chest.anim.meta:4]: also this?

File: New Unity Project/Assets/Animation/Chest.anim.meta [hunk 3]
```diff
@@ -0,0 +1,12 @@
+fileFormatVersion: 2
+<<<<<<< HEAD:New Unity Project/Assets/Animation/Chest.anim.meta
```
Comment (victoriametzler) [inline New Unity Project/Assets/Animation/Chest.anim.meta:2]: what about this?

File: New Unity Project/Assets/Animation/ChestAnimator.controller.meta [hunk 4]
```diff
@@ -0,0 +1,12 @@
+fileFormatVersion: 2
+<<<<<<< HEAD:New Unity Project/Assets/Animation/ChestAnimator.controller.meta
```
Comment (victoriametzler) [inline New Unity Project/Assets/Animation/ChestAnimator.controller.meta:2]: does this merge conflict need to be deleted?

File: New Unity Project/Assets/Animation/Player_Idle.anim.meta [hunk 5]
```diff
@@ -0,0 +1,12 @@
+fileFormatVersion: 2
+<<<<<<< HEAD:New Unity Project/Assets/Animation/Player_Idle.anim.meta
+guid: 01f01bae0c7a448709137cb675e1f7bd
+=======
+guid: 31a4b95657e7543a583393c898842ed4
+>>>>>>> master:New Unity Project/Assets/Tiles/GrassColumn 1.asset.meta
```
Comment (victoriametzler) [inline New Unity Project/Assets/Animation/Player_Idle.anim.meta:6]: does this merge conflict need to be deleted?

File: New Unity Project/Assets/Animation/Player_Idle_Animator.controller.meta [hunk 6]
```diff
@@ -0,0 +1,12 @@
+fileFormatVersion: 2
+<<<<<<< HEAD:New Unity Project/Assets/Animation/Player_Idle_Animator.controller.meta
+guid: 6e5b5ac228809414391f53057e3332e8
+=======
+guid: 67b5a4f36465347c69ae5ba356cc96af
+>>>>>>> master:New Unity Project/Assets/Tiles/GrassColumn 2.asset.meta
```
Comment (victoriametzler) [inline New Unity Project/Assets/Animation/Player_Idle_Animator.controller.meta:6]: what about this?

**Review by isawells — APPROVED** [review]

**Review by victoriametzler — APPROVED** [review]

**Pull request merged**